src/components/molecules/tabbed-navigation/tabbed-navigation.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactElement, useState } from 'react';
+import { TabProps } from './tab';
+
+import './tabbed-navigation.scss';
+
+type TabbedNavigationProps = {
+  initiallySelected?: number,
+  children: ReactElement<TabProps>[],
+};
+
+export const TabbedNavigation = ({ children, initiallySelected = 0 }: TabbedNavigationProps): JSX.Element => {
+  const [activeTab, setActiveTab] = useState(initiallySelected);
+
+  return (
+    <div className="tabbed-navigation">
+      <ul className="tabbed-navigation__tabs">
+        {children.map((child, index) => {
+          const { label } = child.props;
+
+          return (<li className={`tabbed-navigation__tab-label ${activeTab === index ? 'tabbed-navigation__tab-label--active' : ''}`} key={label} onClick={() => setActiveTab(index)}>{label}</li>);
+        })}
+      </ul>
+      {children[activeTab]}
```

Comment:
This should probably be in a container like it was before so we can add things like padding when the content is added. It also makes the content siblings of the tab headers which feels wrong and could have issues with layout.

Reply:
Each <Tab> element creates a `<div class=tabbed-navigation_content />` with it's content within. Children here is a collection of those divs. 

So I think it does do what you're asking, just a level below as part of the <Tab> element that is showing. I think it's probably doable to have <Tab> just splurge it's children one way or another though, if it's easier to read/preferable?

Reply:
Thats true, i missed that. 😁